cmd/loop/quote.go

```diff
@@ -97,6 +98,17 @@ func quote(ctx *cli.Context) error {
 		return err
 	}
 
+	// For loop in, the fee estimation is handed to lnd which tries to
+	// construct a real transaction to sample realistic fees to pay to the
+	// HTLC. If the wallet doesn't have enough funds to create this TX, we
+	// don't want to fail the quote. But the user should still be informed
+	// why the fee shows as 0.
+	if in && quoteResp.MinerFee == 0 {
```

Comment:
Can it be an error code on the quote response?

Reply:
I added a quote status.

Reply:
Removed the status again and went with fee of -1.